scripts/lib/libvirt/libvirt_setup.py

```diff
@@ -56,14 +56,24 @@ def get_machine_arch():
     return os.uname()[4]
 
 
-def get_os_loader(firmware_type=None):
+def get_os_loader(firmware_type=None, secureboot=False):
     path = None
-    template = "<loader readonly='yes' type='pflash'>%s</loader>"
+    template = """<loader readonly='yes' type='pflash'>/usr/share/qemu/{0}</loader>
+{1}"""
+    nvram = "<nvram>/var/lib/libvirt/$cloud-node${nodecounter}_VARS.fd</nvram>"
     if 'aarch64' in get_machine_arch():
-        path = "/usr/share/qemu/aavmf-aarch64-code.bin"
+        path = "aavmf-aarch64-code.bin"
```

Comment:
this just uses the default nvram specified 2 lines above, no need to override it.